### ISSUE TITLE
create a new j.mapArray to create several j.map externals, with their attributes as internal parameters

### ISSUE THREAD

**Issue by hems** [jamoma/JamomaMax#45]
so they can be stored and recalled in cues

 see more on the original redmine issue http://redmine.jamoma.org/issues/1546

**Comment by reno-** [comment]
I assigne this one for you Pascal in order to know if it is still a request, as I know that you use a lot Arrays

**Comment by bltzr** [comment]
Yes, I would still like to have this… though that’s not top priority at all...

**Comment by reno-** [comment]
ok, so I guess it's for @theod
